recipes/install.rb

```diff
@@ -92,6 +92,7 @@
   docroot docroot
   es_server node['kibana']['es_server']
   kibana_port node['kibana']['java_webserver_port']
-  template 'kibana-nginx_file.conf.erb'
+  template node['kibana']['nginx']['template']
```

Comment:
The value of `node['kibana']['nginx']['template']` appears to be `kibana-nginx.conf.erb`, but the original value is `kibana-nginx_file.conf.erb`. I haven't tried to converge this yet, but there isn't a template in this cookbook called `kibana-nginx.conf.erb`. I assume this is not going to converge, right?